### PR TITLE
clean up prior api refs and links

### PR DESCRIPTION
**Pull request by kbhawkey**
- Remove previous API references and links in docs
- Updated several blog pages. Investigating if these links are available (in snapshots) or
if the links should point to the latest version.
- Other minor cleanup

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -339,12 +340,12 @@ audit policies.
 [Fluentd][fluentd] is an open source data collector for unified logging layer.
 In this example, we will use fluentd to split audit events by different namespaces.
 
-1. install [fluentd][fluentd_install_doc],  fluent-plugin-forest and fluent-plugin-rewrite-tag-filter in the kube-apiserver node
-{{< note >}}
-Fluent-plugin-forest and fluent-plugin-rewrite-tag-filter are plugins for fluentd. You can get details about plugin installation from [fluentd plugin-management][fluentd_plugin_management_doc].
+{{< note >}}Fluent-plugin-forest and fluent-plugin-rewrite-tag-filter are plugins for fluentd. You can get details about plugin installation from [fluentd plugin-management][fluentd_plugin_management_doc].
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/audit.md:343]: Looks like some unrelated changes might have crept in here?

Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/audit.md:343]: Yes, I moved the note. The list numbering was broken with the embedded note.
